provider.py

```diff
@@ -45,8 +49,8 @@ def providerKey(cls):
         return 'EE'
 
     @classmethod
-    def createProvider(cls, uri, providerOptions):
-        return EarthEngineRasterDataProvider(uri, providerOptions)
+    def createProvider(cls, uri, *args, **kwargs):
```

Comment:
The arguments are incorrect, this is an overloaded method, see https://qgis.org/pyqgis/3.2/core/Provider/QgsProviderRegistry.html

Reply:
yeah, right, when I tested it show me an error missing an argument I based on QgsProvideRregistry.py file that uses args and kwargs, but I check some test implementation and I found the missing argument is the flags, fixed in a40538f67723ef2d48a90be934152c91e8495d22 based on https://github.com/qgis/QGIS/blob/master/tests/src/python/provider_python.py#L198